### PR TITLE
fix: when editor changes from display:none to visible default ui and …

### PR DESCRIPTION
**Pull request by macrozone**
…resize handles are wrong

fixes https://github.com/react-page/react-page/issues/803

i fixed it by using interactionObserver. This does not work in IE11, but the editor should not be used there anyway (only in readOnly mode).

@PeterKottas I'll merge and release today, i'll guess this small change is ok

### CODE REVIEW

File: tslint.json [hunk 1]
```diff
@@ -66,7 +66,7 @@
     ],
     "quotemark": [true, "single", "jsx-double"],
     "radix": true,
-    "semicolon": [true, "always"],
+    "semicolon": false,
```
Comment (PeterKottas) [inline tslint.json:69]: This is a fairly major change that would probably break a lot of code (in terms of lint). Why do we want to change it here?

**Comment by PeterKottas** [comment]
Just the lint thing, rest looks good.

**Comment by macrozone** [comment]
@PeterKottas I noticed that prettier and tslint are not in sync. but its true, the change is not ok here. we should add the config-prettier plugin (or even switch to eslint as this now supports typescript)

**Comment by macrozone** [comment]
@PeterKottas as no PRs were open, i used the moment to switch to eslint and reformat everything according to the rules, 

i hope its now beetter

**Comment by PeterKottas** [comment]
Good stuff. Then it's all good.

**Pull request merged**